metric/unaggregated/types.go

```diff
@@ -119,14 +123,14 @@ func (m *MetricUnion) String() string {
 	}
 }
 
-// Reset resets the metric union
+// Reset resets the metric union.
```

Comment:
Do we need to check if the metric has pooled timers here and return them if so? Something like:

```
if m.TimerValPool != nil {
  // return floats to pool
  ..
}
*m = emptyMetricUnion
```